### ISSUE TITLE
Escape quotes in config

### ISSUE THREAD

**Issue by kelson42** [openzim/zimfarm#668]
See https://farm.openzim.org/pipeline/a7e335828162de57f03eaf16/debug. It seems the description argument is not quoted properly! Or maybe this is a ticket for Zimfarm?

**Comment by rgaudin** [comment]
Yes, that's not something the scraper can handle because it never reaches it. I transfered to zimfarm.

Although we haven't seen those, there are certainly legitimate use cases for this and it needs to be fixed but for this very recipe, it's not correct. I fixed it by using `«Comment faire»`. Even for English, `"` is not appropriate ; `“` and `”` are.

**Comment by elfkuzco** [comment]
@benoit74 , can you take a look at this? Seems to have been fixed as a consequence of the `zimfarm-upgrade`

**Comment by benoit74** [comment]
I confirm it is fixed: a description like `Research' a"nd test low-tech solutions, and document findings` is now causing a command-line argument like `--description='Research'"'"' a"nd test low-tech solutions, and document findings'`.

Which is correct (on Linux systems at least):

```
> echo 'Research'"'"' a"nd test low-tech solutions, and document findings'
Research' a"nd test low-tech solutions, and document findings
```

And UI still display unescaped description everywhere (besides in command line obviously).

This will be deployed in prod when `zimfarm-upgrade` branch is deployed.
